src/apply.rs

```diff
@@ -4,11 +4,11 @@ use kube::ResourceExt;
 pub const KUBIT_APPLIER_FIELD_MANAGER: &str = "kubit-applier";
 
 /// Generates shell script that will apply the manifests and writes it to w
-pub fn emit_script<W>(app_instance: &AppInstance, w: &mut W) -> Result<()>
+pub fn emit_script<W>(app_instance: &AppInstance, is_local: bool, w: &mut W) -> Result<()>
```

Comment:
Introducing this `is_local` in a bunch of places feels a bit heavy-handed, but my intention is to get it something working first and then refactor from a known good point.